core/src/main/java/org/infinispan/commands/read/KeySetCommand.java

```diff
@@ -48,8 +48,8 @@ public Object acceptVisitor(InvocationContext ctx, Visitor visitor) throws Throw
    }
 
    @Override
-   public boolean readsExistingValues() {
-      return false;
+   public LoadType loadType() {
+      return null;
```

Comment:
I had forgotten that you already have `DONT_LOAD`, and I thought that's what `null` was supposed to mean.